packages/server/api/src/app/database/migrations/1749644220381-AddStepIdToExistingFlowVersionsAndInsertOutputInTestOutputTable.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { encryptUtils, fileCompressor, logger } from '@openops/server-shared';
+import { FileCompression, openOpsId } from '@openops/shared';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddStepIdToExistingFlowVersionsAndInsertOutputInTestOutputTable1749644220381
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    logger.info(
+      'AddStepIdToExistingFlowVersionsAndInsertOutputInTestOutputTable1749644220381: starting',
+    );
+
+    const workflows = await queryRunner.query(
+      'SELECT "id", "trigger" FROM "flow_version"',
+    );
+
+    await updateRecords(queryRunner, workflows, 'flow_version');
+
+    logger.info(
+      'AddStepIdToExistingFlowVersionsAndInsertOutputInTestOutputTable1749644220381: completed',
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    throw new Error('Rollback not implemented');
+  }
+}
+
+async function updateRecords(
+  queryRunner: QueryRunner,
+  records: { id: string; trigger: any }[],
+  tableName: string,
+): Promise<void> {
+  for (const record of records) {
+    const jsonData = record.trigger;
+
+    const { updatedJson, extractedData } = await processJsonObject(jsonData);
+
+    await queryRunner.query(
+      `UPDATE "${tableName}" SET "trigger" = $1 WHERE "id" = $2`,
+      [updatedJson, record.id],
+    );
+
+    for (const data of extractedData) {
+      const encryptOutput = encryptUtils.encryptObject(
```

Comment:
I think we don't need this @MarceloRGonc , we're only patching the step ids, no?